tools/matinfo/src/main.cpp

```diff
@@ -132,24 +137,26 @@ static void printUsage(const char* name) {
     std::string execName(utils::Path(name).getName());
     std::string usage(
             "MATINFO prints information about material files compiled with matc\n"
-                    "Usage:\n"
-                    "    MATINFO [options] <material file>\n"
-                    "\n"
-                    "Options:\n"
-                    "   --help, -h\n"
-                    "       Print this message\n\n"
-                    "   --print-glsl=[index], -g\n"
-                    "       Print GLSL for the nth shader (0 is the first OpenGL shader)\n\n"
-                    "   --print-spirv=[index], -s\n"
-                    "       Print disasm for the nth shader (0 is the first Vulkan shader)\n\n"
-                    "   --print-metal=[index], -m\n"
-                    "       Print Metal Shading Language for the nth shader (0 is the first Metal shader)\n\n"
-                    "   --print-vkglsl=[index], -v\n"
-                    "       Print the nth Vulkan shader transpiled into GLSL\n\n"
-                    "   --dump-binary=[index], -b\n"
-                    "       Dump binary SPIRV for the nth Vulkan shader to 'out.spv'\n\n"
-                    "   --license\n"
-                    "       Print copyright and license information\n\n"
+            "Usage:\n"
+            "    MATINFO [options] <material file>\n"
+            "\n"
+            "Options:\n"
+            "   --analyze-spirv=[index], -a\n"
```

Comment:
Can you move this option at the end of the help message? It's really for us and shouldn't come first (especially not before the more common options)

Reply:
Fixed.